<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>glav</class>
 <widget class="QMainWindow" name="glav">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>813</width>
    <height>584</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>expense analyzer</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 0, 255);
font: 8pt &quot;MS Shell Dlg 2&quot;;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_7">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="styleSheet">
         <string notr="true">font: 25pt &quot;MS Shell Dlg 2&quot;;
color: rgb(0, 255, 255);</string>
        </property>
        <property name="text">
         <string>Добро пожаловать!</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QFrame" name="all">
        <property name="styleSheet">
         <string notr="true">border: 1px solid rgb(255, 85, 255);</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QFrame" name="balance_1">
           <property name="styleSheet">
            <string notr="true">border: 1 px solid rgba(255, 85, 255, 35);
border-radius: 8px;</string>
           </property>
           <layout class="QVBoxLayout" name="balance">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout">
              <item>
               <widget class="QLabel" name="label_3">
                <property name="styleSheet">
                 <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(200, 0, 235);
font: 75 16pt &quot;MS Shell Dlg 2&quot;;
border: none;</string>
                </property>
                <property name="text">
                 <string>Баланс:</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="balance_all">
                <property name="styleSheet">
                 <string notr="true">color: rgb(0, 255, 255);
font: 24pt &quot;MS Shell Dlg 2&quot;;
border: none;</string>
                </property>
                <property name="text">
                 <string>11</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QGridLayout" name="gridLayout_2">
              <item row="1" column="0">
               <widget class="QLabel" name="label_6">
                <property name="styleSheet">
                 <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(200, 0, 235);
font: 14pt &quot;MS Shell Dlg 2&quot;;
border: none;</string>
                </property>
                <property name="text">
                 <string>Доход:</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QLabel" name="label_4">
                <property name="styleSheet">
                 <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(200, 0, 235);
font: 14pt &quot;MS Shell Dlg 2&quot;;
border: none;</string>
                </property>
                <property name="text">
                 <string>Расход:</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="up_blnc">
                <property name="styleSheet">
                 <string notr="true">color: rgb(0, 255, 255);
font: 20pt &quot;MS Shell Dlg 2&quot;;
border: none;</string>
                </property>
                <property name="text">
                 <string>11</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QLabel" name="down_blnc">
                <property name="styleSheet">
                 <string notr="true">color: rgb(0, 255, 255);
font: 20pt &quot;MS Shell Dlg 2&quot;;
border: none;</string>
                </property>
                <property name="text">
                 <string>11</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="expense">
           <property name="styleSheet">
            <string notr="true">border: 1 px solid rgba(255, 85, 255, 35);
border-radius: 8px;</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QLabel" name="label_8">
              <property name="styleSheet">
               <string notr="true">color: rgb(0, 255, 255);
background-color: rgb(200, 0, 235);
font: 75 16pt &quot;MS Shell Dlg 2&quot;;
border: none;</string>
              </property>
              <property name="text">
               <string>Категории расходов:</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QGridLayout" name="gridLayout_3">
              <item row="2" column="1">
               <widget class="QLabel" name="transport_blnc">
                <property name="styleSheet">
                 <string notr="true">color: rgb(0, 255, 255);
font: 75 11pt &quot;MS Shell Dlg 2&quot;;
border: none;</string>
                </property>
                <property name="text">
                 <string>11</string>
                </property>
               </widget>
              </item>
              <item row="3" column="1">
               <widget class="QLabel" name="other_blnc">
                <property name="styleSheet">
                 <string notr="true">color: rgb(0, 255, 255);
font: 75 11pt &quot;MS Shell Dlg 2&quot;;
border: none;</string>
                </property>
                <property name="text">
                 <string>11</string>
                </property>
               </widget>
              </item>
              <item row="3" column="0">
               <widget class="QLabel" name="label_12">
                <property name="styleSheet">
                 <string notr="true">color: rgb(170, 255, 255);
background-color: rgb(200, 0, 245);
font: 75 11pt &quot;MS Shell Dlg 2&quot;;
border: none;</string>
                </property>
                <property name="text">
                 <string>Другое (досуг и т.д.)</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QLabel" name="grsries_clothes_blnc">
                <property name="styleSheet">
                 <string notr="true">color: rgb(0, 255, 255);
font: 75 11pt &quot;MS Shell Dlg 2&quot;;
border: none;</string>
                </property>
                <property name="text">
                 <string>11</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QLabel" name="credit_expense">
                <property name="maximumSize">
                 <size>
                  <width>100</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true">color: rgb(0, 255, 255);
font: 75 11pt &quot;MS Shell Dlg 2&quot;;
border: none;</string>
                </property>
                <property name="text">
                 <string>11</string>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="label_11">
                <property name="styleSheet">
                 <string notr="true">color: rgb(170, 255, 255);
background-color: rgb(200, 0, 245);
font: 75 11pt &quot;MS Shell Dlg 2&quot;;
border: none;</string>
                </property>
                <property name="text">
                 <string>Транспорт</string>
                </property>
               </widget>
              </item>
              <item row="0" column="0">
               <widget class="QLabel" name="label_9">
                <property name="styleSheet">
                 <string notr="true">color: rgb(170, 255, 255);
background-color: rgb(200, 0, 245);
font: 75 11pt &quot;MS Shell Dlg 2&quot;;
border: none;</string>
                </property>
                <property name="text">
                 <string>Кредиты, налоги</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="label_10">
                <property name="styleSheet">
                 <string notr="true">color: rgb(170, 255, 255);
background-color: rgb(200, 0, 245);
font: 75 11pt &quot;MS Shell Dlg 2&quot;;
border: none;</string>
                </property>
                <property name="text">
                 <string>Продукты питания, промтовары, одежда</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_17">
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(200, 0, 215);
font: 75 16pt &quot;MS Shell Dlg 2&quot;;
border: none;</string>
        </property>
        <property name="text">
         <string>Транзакция</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="btn_add_trnsctn">
          <property name="styleSheet">
           <string notr="true">QPushButton {
color: rgb(0, 255, 255);
background-color: rgb(200, 0, 235);
font: 14pt &quot;MS Shell Dlg 2&quot;;
border: 1px solid rgb(200, 0, 245);
border-radius: 7px;
width: 200 px;
height: 45px;
}
QPushButton:hover {
background-color: rgb(255, 129, 245);
}
QPushButton:pressed {
background-color: rgba(200, 0, 245, 65);
}</string>
          </property>
          <property name="text">
           <string>Новая</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_remove_trnsctn">
          <property name="styleSheet">
           <string notr="true">QPushButton {
color: rgb(0, 255, 255);
background-color: rgb(200, 0, 235);
font: 14pt &quot;MS Shell Dlg 2&quot;;
border: 1px solid rgb(200, 0, 245);
border-radius: 7px;
width: 200 px;
height: 45px;
}
QPushButton:hover {
background-color: rgb(255, 129, 245);
}
QPushButton:pressed {
background-color: rgba(200, 0, 245, 65);
}</string>
          </property>
          <property name="text">
           <string>Удалить</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_5"/>
    </item>
    <item>
     <widget class="QTableView" name="tableView">
      <property name="styleSheet">
       <string notr="true">QTableView {
background-color: rgba(0, 255, 255, 25);
border: 1px solid rgba(0, 255, 255, 50);
border-bottom-right-radius: 8px;
border-bottom-left-radius: 8px;
}

QTableView: section {
background-color: rgb(0, 255, 255);
color: rgb(255, 0, 0);
border: none;
height: 45px;
font: 14pt &quot;MS Shell Dlg 2&quot;;
}

QTableView: item {
border-style: none;
border-bottom: rgba(0, 255, 255, 45);
}

QTableView: item:selected {
border: none;
color: rgb(255, 0, 0);
background-color: rgba(0, 255, 255, 45);
}</string>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="showGrid">
       <bool>false</bool>
      </property>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>133</number>
      </attribute>
      <attribute name="horizontalHeaderMinimumSectionSize">
       <number>43</number>
      </attribute>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>813</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
